Rlease v5.0.0 (#2630)
theupdateframework/python-tuf: docs/CHANGELOG.md

# Changelog

## v5.0.0

This release, most notably, marks stable securesystemslib v1.0.0 as minimum
requirement. The update causes a minor break in the new DSSE API (see below)

> and might affect usage of the Metadata API. See the [securesystemslib release

Reviewer: ```suggestion
and affects users who also directly depend on securesystemslib. See the [securesystemslib release
```
